radixdlt-java/src/test/java/com/radixdlt/client/application/translate/tokenclasses/TokenReducerTest.java

```diff
@@ -24,11 +27,14 @@ public void testTokenWithNoMint() {
 		when(tokenParticle.getName()).thenReturn("Name");
 		when(tokenParticle.getSymbol()).thenReturn("ISO");
 		when(tokenParticle.getDescription()).thenReturn("Desc");
+		when(tokenParticle.getTokenPermissions()).thenReturn(Collections.singletonMap(FungibleType.MINTED, TokenPermission.SAME_ATOM_ONLY));
 
 		TokenReducer tokenReducer = new TokenReducer();
 		Map<TokenClassReference, TokenState> state = tokenReducer.reduce(Collections.emptyMap(), SpunParticle.up(tokenParticle));
 		assertThat(state.get(tokenRef))
-				.isEqualToComparingFieldByField(new TokenState("Name", "ISO", "Desc", BigDecimal.ZERO));
+			.isEqualToComparingFieldByField(
```

Comment:
Could just use `assertEquals(a, b)` now that `TokenState` has an `equals(...)` method.